xprof/xprof.rb.in

```diff
@@ -1038,7 +1038,9 @@ if __FILE__ == $PROGRAM_NAME
 
   if mpi_master?
     warn("THAPI: Trace location: #{folder}")
-    XprofExitCode.update(gm_processing(folder), 'babeltrace_thapi')
+    if OPTIONS[:analysis]
```

Comment:
Not sure if this check should be done [here](https://github.com/argonne-lcf/THAPI/pull/410/commits/7d721cb2a0063c671ca4aeea049088cfc071fb60#diff-91764bbba95858e5daeb032846d78ac2bc0e322c2e0530cecafdc97d0808203cL893).

Reply:
Oh maybe! We always need to put the medatada but maybe `gm_processing` have not been guarded with `no-analysis`
(sorry in a meeting fulll day for the 2 next day)